### ISSUE TITLE
Intel MKL Error on Python 3.5 Travis Builds

### ISSUE THREAD

**Issue by jlnav** [Libensemble/libensemble#376]
```
INTEL MKL ERROR: /home/travis/miniconda/envs/condaenv/lib/libmkl_avx512.so: undefined symbol: mkl_sparse_optimize_bsr_trsm_i8.

Intel MKL FATAL ERROR: Cannot load libmkl_avx512.so or libmkl_def.so.
```

The above causes Test 86 (test_persistent_tasmanian.py, I think) to fail on Python 3.5 builds.

**Comment by jmlarson1** [comment]
This is an intermittent error on Travis, so I wouldn't spend too much time on it at this time. Also, 3.5 reaches end-of-life status in the fall.

**Comment by jmlarson1** [comment]
This doesn't seem intermittent anymore. A google shows that possibly rebuilding numpy will help:

https://github.com/ContinuumIO/anaconda-issues/issues/720

Thoughts?

**Comment by jlnav** [comment]
I tried reproducing the Travis builds in a docker container with all the same dependencies. Two odd things: 1) I think everything worked perfectly. 2) MKL wasn't installed. 

Are there alternatives to MKL that could replace it on Travis?

**Comment by jmlarson1** [comment]
MKL is not our dependency, but Tasmanian. I'm not sure we want to spend time switch their dependencies.

**Comment by jlnav** [comment]
Tentatively closed by #381 ?
